modules/statics/src/coins.ts

```diff
@@ -802,6 +804,7 @@ export const coins = CoinMap.fromCoins([
   erc20('wcfg', 'Wrapped Centrifuge', 18, '0xc221b7e65ffc80de234bbb6667abdd46593d34f0', UnderlyingAsset.WCFG),
   erc20('wec', 'Whole Earth Coin ', 18, '0xcc1a8bd438bebc4b2a885a34475bb974f2124317', UnderlyingAsset.WEC),
   erc20('wet', 'We Show Token', 18, '0x36d10c6800d569bb8c4fe284a05ffe3b752f972c', UnderlyingAsset.WET),
+  erc20('weth', 'Wrapped ETH', 18, '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2', UnderlyingAsset.WETH),
```

Comment:
```suggestion
  erc20('weth', 'Wrapped Ether', 18, '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2', UnderlyingAsset.WETH),
```

Reply:
same comment as above

Reply:
This is correct; confirmed that "Wrapped Ether" is correct